Things.md

```diff
@@ -0,0 +1,139 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are manipulated. Everything you see in the game (trees, blocks, the player, etc.) is a Thing. The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their properties, placements, etc.    
+  
+### Table of Contents
+1. [Manipulating Things](#manipulating-things)
+    1. [Shifting Things](#shifting-things)
+        1. [Shift](#shift)
+        2. [Set](#set)
+        3. [Midpoints](#midpoints)
+        4. [Update](#update)
+        5. [Slide](#slide)
+    2. [Opacity](#opacity)
+2. [GroupHoldr](#groupholdr)
+    1. [Getting Groups](#getting-groups)
+
+# Manipulating Things
+
+## Shifting Things  
+A Thing's placement on the screen is based on its `top`, `right`, `bottom`, and `left` properties. These define the bounding box for a Thing. A Thing is basically just a rectangle, with its height, width, and location defined by these properties. The coordinate system in which these Things are placed has an origin at the top left corner (0,0). Things are also scaled from these raw measurements to actual in-game measurements using a predefined `unitsize`. There are a number of ways to move Things, all of which are done using EightBitter and by manipulating these 4 properties.  
+
+#### Shift 
+The first is to use the shift functions; `shiftVert` and `shiftHoriz`. These move the Thing up and down or left and right.
+```javascript
+// Shifting a thing Vertically by 1.
+shiftVert(thing, 1);
+
+// Shifting a thing Horizontally by 1. 
+shiftHoriz(thing, 1);
+```
+#### Set 
+The next is to use a `set` function (`setTop`, `setRight`, `setBottom`, or `setLeft`). These move the Thing to line up with the bounding property.
+```javascript
+// Setting the top bound of a Thing to 10. 
+setTop(thing, 10);
+
+// Setting the left bound of a Thing to 10. 
+setLeft(thing, 10);
+
+// Setting the bottom bound of a Thing to 10. 
+setBottom(thing, 10);
+
+// Setting the right bound of a Thing to 10. 
+setRight(thing, 10);
+```  
+#### Midpoints
+The next is to set the Thing's midpoint using `setMidX`, `setMidY`, and `setMid` (the latter of which sets both the `x` and `y` midpoints). These functions make it so the Thing is centered on the given `x` and `y`.  
+```javascript
+// Setting both x and y midpoints to 1 and 1, respectively. 
+setMid(thing, 1, 1);
+
+// The above is equivalent to the following 2 function calls. 
+setMidX(thing, 1);
+setMidY(thing, 1);
+```
+Going along with setting midpoints, we can do that based on the midpoint of another Thing. Using `setMidXObj`, `setMidYObj`, and `setMidObj` you can shift a Thing so that its horizontal and/or vertical midpoint(s) are centered on that of another Thing.  
+```javascript
+/* Aligning a Thing's horizontal and vertical midpoints with
```

Comment:
This should be two 1-line comments.

I'm very against using `/* */` style syntax for simple explanations. IMO, it should be reserved for block-level comments (JSDoc).